contracts/lido/LidoInsuranceTest.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "../AnteTest.sol";
+
+contract AnteLidoInsuranceTest is AnteTest("Make sure at least 0.5% of Lido stake is insured!") {
+    address public immutable lidoContract;
+    address public immutable lidoInsuranceContract;
+
+    constructor (address _lidoContract, address _lidoInsuranceContract) {
+        lidoContract = _lidoContract;
+        lidoInsuranceContract = _lidoInsuranceContract;
+        protocolName = "Lido Staking ";
+        testedContracts = [0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84];
```

Comment:
Since this is the same address as the lidoContract, and that you're testing both the lido contract and insurance contract. I would say put those two in the testedContracts here
```
testedContracts = [_lidoContract, _lidoInsuranceContract];
```
As these are the contracts that we're looking at directly.